products/prisma-cloud/docs/cspm/AWS-cloud-account-onboarding.md

```diff
@@ -0,0 +1,550 @@
+---
+id: aws-cloud-account-onboarding
+title: AWS Cloud Account Onboarding on to Prisma Cloud using Automation
+---
+
+This guide shows how to automate AWS Cloud Account onboarding on Prisma Cloud.
+
+To successfully onboard an AWS account in Prisma Cloud, you must create an IAM Role in your AWS account that has a trusted relationship with Prisma Cloud AWS Account. Prisma Cloud uses IAM Role ARN to *AssumeRole* in to your AWS account and ingest various configurations and logs.
+
+*[Recommended]* To Onboard the AWS cloud account using *Terraform*, Follow
+[AWS organization](https://github.com/PaloAltoNetworks/terraform-provider-prismacloud/tree/master/docs/resources/org_cloud_account_v2.md) or [AWS account](https://github.com/PaloAltoNetworks/terraform-provider-prismacloud/tree/master/docs/resources/cloud_account_v2.md) guides.
+
+To Onboard using other automation tools(such as python, etc), follow the steps listed below:
+
+
+***Prerequisite***: Obtain an authorization token by [Logging In](/prisma-cloud/api/cspm/app-login/) ![alt text](/icons/api-icon-pan-dev.svg)
+
+:::info
+
+- The token is only valid for 10 minutes, so [refresh it](/prisma-cloud/api/cspm/extend-session/) ![alt text](/icons/api-icon-pan-dev.svg) as needed if you believe your workflow might take longer than that.
+
+- The base url in this guide is a generic `api.prismacloud.io`. Replace it with yours accordingly.
+
+:::
+
+### The steps to Onboard AWS Account/Organization are outlined below:
+
+1. Get the *Supported Features* for the cloud type.
+2. Generate *CFT* and *External ID* using Prisma Cloud *CFT Generation API* and create *IAM Role* using the generated CFT.
+3. Use Prisma Cloud *Save API* to onboard the account by passing the *IAM Role ARN* and other parameters in the request payload.
+
+![](/img/aws-cloud-account-onboarding-workflow-automation.png)
+
+
+
+### 1. Fetch *Supported Features* for cloud type and account type
+
+  To get the list of supported features, call [Fetch Supported Features](/prisma-cloud/api/cspm/fetch-supported-features/) ![alt text](/icons/api-icon-pan-dev.svg) with the required payload.<br/>
+
+  The response contains *supportedFeatures* key whose value contains supported feature names.
```

Comment:
wrape supportedFeatures into ticks `supportedFeatures` because it's a property and shows as code snippet